lib/game_engine.rb

```diff
@@ -0,0 +1,15 @@
+
+class GameEngine
+    attr_reader :token
+    def initialize(token)
+        @token = token
+    end
+
+    def run_game
+    end
```

Comment:
^Not being used yet, so shouldn't be apart of the PR.

Reply:
Got it! I'll update the PR